packages/office-ui-fabric-react/package.json

```diff
@@ -22,6 +22,7 @@
     "@types/enzyme": "^2.7.5",
```

Comment:
It's easy. 

```base
npm i -g @microsoft/rush
rush change -b upstream/master 
```

Then answer questions. "upstream/master" refers to the OfficeDev remote master branch, you may need to register "upstream" pointing to the OfficeDev remote.